charts/cryostat/templates/storage_secret.yaml

```diff
@@ -1,7 +1,9 @@
+{{- if empty .Values.storage.storageSecretName -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: {{ .Release.Name }}-storage
+  name: {{ .Release.Name }}-storage-secret
 type: Opaque
 data:
-  SECRET_KEY: {{ include "cryostat.objectStorageSecretKey" . }}
+  STORAGE_ACCESS_KEY: {{ include "cryostat.objectStorageSecretKey" . }}
```

Comment:
Just a question tho: Changing the secret name or secret key name here means when upgrading to new chart version, this would break the existing connection to S3 storage instance?

Reply:
I guess if its not used for encrypting data in S3, then it should be fine?

Reply:
Yes, it would. This next release is billed as a new major version (4.0) with other breaking changes anyway. Hopefully after this one we will have only minor semver release versions and nice upgrade paths for quite a long time.

Reply:
Oh right sounds good! Thanks!!

Reply:
Right, this secret isn't used for encrypting the data at rest in S3, it's basically the service account password for authenticating to S3.

Reply:
ohh then it should not break the connection the. I haven't checked tho but will keep an eye for that :D